docs/models.yml

```diff
@@ -23,8 +23,8 @@
 #       `motion/category_id`. The type indicates that there are many
```

Comment:
Did not check everything again, only the lines where I put a remark. If you changed something else, please ping me again ;)